01/index.js

```diff
@@ -0,0 +1,25 @@
+function tree(jsonObj, prefix = "", isFirst = true, isLast = true) {
+  console.log(
+    prefix + (isFirst ? "├── " : isLast ? "└── " : "├── ") + jsonObj.name
```

Comment:
oough 2 ternary operators in one `console.log` 😅

Reply:
That's how you make a good obfuscation, lol